src/koaApi.coffee

```diff
@@ -77,10 +79,27 @@ exports.setupApp = (done) ->
   app.use route.get '/visualizer/events/:receivedTime', visualizer.getLatestEvents
   app.use route.get '/visualizer/sync', visualizer.sync
 
+# -- Old metrics Routes for Backward Compatibility
   app.use route.get '/metrics', metrics.getGlobalLoadTimeMetrics
   app.use route.get '/metrics/status', metrics.getGlobalStatusMetrics
   app.use route.get '/metrics/:type/:channelId', metrics.getChannelMetrics
-  
+# -- End of Old Metrics Routes --
```

Comment:
Could we not rather just replace the metrics endpoints above with with a switch between the old metrics and the new statsd functions as you have done below? They should produce the data in the same format right? Then you could do away with the `/stats/*` routes and just include some extra routes for the statsd specific routes. Any reason why you can't do that?